app/assets/stylesheets/content/_tables.sass

```diff
@@ -26,14 +26,59 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
+table
+  td
+    padding: 3px 6px
+  p
+    margin: 0
+
+  &.files tr.file td
+    text-align: center
+    &.filename
+      text-align: left
+      padding-left: 24px
+    &.digest
+      font-size: 80%
+
+  &.members td.roles, &.memberships td.roles
+    width: 45%
+
+  &.plugins
+    td
+      vertical-align: middle
+      &.configure
+        text-align: right
+        padding-right: 1em
+    span
+      &.name
+        font-weight: bold
+      &.description, &.url
+        display: block
+        font-size: 0.9em
+
+  &#time-report
+    tbody tr
+      font-style: italic
+      color: #333
+      &.last-level
+        font-style: normal
+        color: #555
+      &.total
+        font-style: normal
+        font-weight: bold
+        color: #555
+        background-color: #EEEEEE
+    .hours-dec
+      font-size: 0.9em
```

Comment:
I am not sure what the intention of the different `.sass` files is.

Therefore I am yet unsure, whether such special rules (the ones above only apply to http://localhost:5000/projects/1/time_entries/report) belong into this file (because they style things around a table) or into a separate file (e.g. `_time_report.sass`). Since the `.hours-dec` rule even applies to an element outside a table, I'd vote for the latter.

On an unrelated note: most of the rules above are quite useless, I don't think that they add any value to the table:

* the difference between `#333` and `#555` is too subtle to notice
* on the other hand `#555` has quite a bad contrast on `#eee`

If I had to decide, I would just kill those rules and not care if they belong here or elsewhere.